### PR TITLE
Make badgeNone usable in HTML img tag (returning 200 instead of 404)

### PR DESCRIPTION
**Pull request by strk**
Also adds a print in the log about the kind of failure

### CODE REVIEW

**Comment by tboerger** [comment]
You should use `logrus` like it's done at https://github.com/drone/drone/blob/master/server/build.go#L8 and https://github.com/drone/drone/blob/master/server/build.go#L156 otherwise it's :-1: for me.

File: server/badge.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package server
 
 import (
 	"fmt"
+	log "github.com/Sirupsen/logrus"
```
Comment (tboerger) [inline server/badge.go]: Just a tiny nitpick, put that one line below with a blank line separated from the core imports.

**Comment by tboerger** [comment]
LGTM

File: server/badge.go [hunk 2]
```diff
@@ -42,7 +43,8 @@ func GetBadge(c *gin.Context) {
 
 	build, err := store.GetBuildLast(c, repo, branch)
 	if err != nil {
-		c.String(404, badgeNone)
+		log.Warning(err)
```
Comment (bradrydzewski) [inline server/badge.go]: actually I'll just adjust this later. I need to go through an audit logging anyway and I don't want to delay the PR :)

**Pull request merged**